inc/TRestGeant4Metadata.h

```diff
@@ -337,6 +337,14 @@ class TRestGeant4Metadata : public TRestMetadata {
         return fRemoveUnwantedTracksVolumesToKeep.count(volumeName) > 0;
     }
 
+    inline std::vector<std::string> GetRemoveUnwantedTracksVolumesToKeep() const {
```

Comment:
What a method name!

Reply:
> What a method name!

🤣 it's just for completeness, its only gonna be used for debugging pourposes probably, thats why I needed it anyway.

Can you approve?

Reply:
It is strange name, do you want to keep or to remove?

Reply:
I approve but the method should be documented for sure, and then probably a more meaning full  name given.

Reply:
I also think a method returning a boolean could be just implemented as

```
Bool_t RemoveUnwantedTracks() { return fRemoveUnwantedTracks;} 
```

So that when you implement it in an `if` condition is more like reading text: `if (RemoveUnwantedTracks() ) ...`

Reply:
> I also think a method returning a boolean could be just implemented as
> 
> ```
> Bool_t RemoveUnwantedTracks() { return fRemoveUnwantedTracks;} 
> ```
> 
> So that when you implement it in an `if` condition is more like reading text: `if (RemoveUnwantedTracks() ) ...`

This method exists, its called `GetRemoveUnwantedTracks`.